docs/content/documentation/systems/Materials/tensor_mechanics/ComputeElasticityTensor.md

```diff
@@ -1,10 +1,181 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
-# ComputeElasticityTensor
+# Compute Elasticity Tensor
 !syntax description /Materials/ComputeElasticityTensor
 
+## Description
+The material `ComputeElasticityTensor` builds the elasticity (stiffness) tensor with various user-selected material symmetry options.
```

Comment:
stiffness -> 'material stiffness'?

Reply:
Isn't stiffness (and inversely compliance) the correct name for the tensor on it's own? Although I could definitely write material elasticity (material stiffness) if that would be clearer for users